api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -506,6 +507,51 @@ message HttpConnectionManager {
   //
   // 3. Tracing decision (sampled, forced, etc) is set in 14th byte of the UUID.
   RequestIDExtension request_id_extension = 36;
+
+  // Configuration of local reply returned by Envoy. If Envoy needs to reject a request due to some
+  // error conditions, it calls sendLocalRely with a http status code and detail error text. If this
+  // config is not specified, the error text will be send as response body in plain text format for
+  // http, in grpc-message header for grpc. This config supports rewritting the status code and error
+  // text. The new error text can be combined from many access log operators, such as %REP(PATH)%.
+  // Its format can be either plain text or JSON. The new error text will be send as response body in
+  // either plain/text or application/json format for http, and in grpc-message header for grpc.
```

Comment:
nit: this part doesn't really belong here, it should probably be on top of LocalReplyConfig.